runtime/onert/core/src/compiler/train/TrainingCompiler.h

```diff
@@ -73,7 +73,7 @@ class TrainingCompiler : public ICompiler
 private:
   std::shared_ptr<ir::Model> _model;
   CompilerOptions *_options;
-  const TrainingInfo *_training_info;
+  const TrainingInfo &_training_info;
```

Comment:
If you want to copy `TrainableInfo`
```suggestion
  const TrainingInfo _training_info;
```